aten/src/ATen/native/cuda/ForeachTernaryOp.cu

```diff
@@ -194,15 +194,18 @@ std::vector<at::Tensor> foreach_tensor_lerp_scalarlist_cuda(
       "foreach_tensor_lerp_scalarlist_cuda",
       [&]() {
         using opmath_t = typename at::opmath_type<scalar_t>;
-        multi_tensor_apply<3, opmath_t>(
-            tensor_lists,
-            scalars,
-            TernaryOpScalarListFunctor<
-                scalar_t,
-                /* depth */ 3,
-                /* r_args_depth */ 2,
-                /* res_arg_index */ 2>(),
-            LerpFunctor<opmath_t>());
+        DISPATCH_MULTI_TENSOR_APPLY([&]() {
```

Comment:
Just a heads up that the 32KB change has been reverted due to some issues. Sorry!